Nexus.Sdk.Shared/ErrorHandling/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.Text.Json;
+using Nexus.Sdk.Shared.Responses;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Nexus.Sdk.Shared.ErrorHandling;
+
+public class CustomExceptionHandler : IExceptionHandler
```

Comment:
Small suggestion just to make it easier for clients/users to know what type of exception handler it is, can you please rename it to `CustomerHttpExceptionHandler` so that they are aware this will provide a custom HTTP error and not something else?

Reply:
Resolved

Reply:
Apologies I see I made a spelling mistake in my last comment. Could you please rename it to `CustomHttpExceptionHandler`? As it is not only Customer specific

Reply:
Aaah, yea I should have double checked as well. Changed :)